spec/spec_helper.rb

```diff
@@ -5,5 +5,6 @@
 require_relative '../lib/ship'
 require_relative '../lib/cell'
 require_relative '../lib/board'
+require './lib/board'
```

Comment:
This require shouldn't be necessary because we are already requiring the board file in the `require_relative` line above